specs/phase1/beacon-chain.md

```diff
@@ -617,10 +631,11 @@ def get_start_shard(state: BeaconState, slot: Slot) -> Shard:
     else:
         # Previous epoch
         shard_delta = get_committee_count_delta(state, start_slot=slot, stop_slot=current_epoch_start_slot)
-        max_committees_per_epoch = MAX_COMMITTEES_PER_SLOT * SLOTS_PER_EPOCH
+        max_committees_per_slot = active_shard_count
+        max_committees_in_span = max_committees_per_slot * (current_epoch_start_slot - slot)
         return Shard(
             # Ensure positive
-            (state.current_epoch_start_shard + max_committees_per_epoch * active_shard_count - shard_delta)
+            (state.current_epoch_start_shard + max_committees_in_span - shard_delta)
```

Comment:
`max_committees_per_epoch * active_shard_count` this didn't actually handle it! if the span of epochs to look back was more than `active_shard_count` then it would underflow

I demonstrated this to myself with the unit test I added